_dev/apps/ui/src/components/product-feed/settings/product-selection/product-selection.vue

```diff
@@ -398,23 +398,21 @@ export default defineComponent({
     await this.$store.dispatch(`productFeed/${ActionsTypes.GET_SHOPS_PRODUCTS_INFOS}`);
 
     // get data from localstorage > store OR create new empty filter
-    const localStorageFilters = getDataFromLocalStorage(localStorageName);
+    const localFilters = getDataFromLocalStorage(localStorageName)
+      || (this.$store.getters[`productFeed/${GetterTypes.GET_PRODUCT_FILTER}`]?.length
+      ?? this.$store.getters[`productFeed/${GetterTypes.GET_PRODUCT_FILTER}`]);
```

Comment:
Not sure the nullish operator is useful here. If I get it properly, when `this.$store.getters[`productFeed/${GetterTypes.GET_PRODUCT_FILTER}`]?.length` is null, at the end `localFilters` will be null as well. For any number returned by the length, you'll return the getter.

It seems you can return the getter without additional check.

Reply:
yes you have right thank you